src/lucky/json_body_parser.cr

```diff
@@ -0,0 +1,21 @@
+# :nodoc:
+class Lucky::JsonBodyParser
+  getter request : HTTP::Request
+
+  def initialize(@request)
+  end
+
+  def parsed_json : JSON::Any
+    if body.blank?
+      JSON.parse("{}")
```

Comment:
instead of firing up the parser could this just be initialized.
```suggestion
      JSON::Any.new(nil)
```

Reply:
Fantastic idea!

Reply:
I agree. Maybe instead of `nil` though, we do `JSON::Any.new({} of String => JSON::Any)`?